conduct.md

```diff
@@ -77,6 +76,22 @@ those projects for enforcement. If you wish to use this code of conduct for your
 own project, consider explicitly mentioning your moderation policy or making a
 copy with your own moderation policy so as to avoid confusion.
 
+### Contact
+
+For CoC-related questions or to report possible violations on the channels
+listed above,
+
+* contact one of the moderators active on that channel if you can identify
+  them, or
+* send an e-mail to [Scala-CoC@epfl.ch](mailto:Scala-CoC@epfl.ch), which
+  forwards to general moderators at the Scala Center, or
+* send an e-mail to one of the following general moderators if for some reason
```

Comment:
I might get rid of the 'for some reason'. thoughts?

```suggestion
* send an e-mail to one of the following general moderators if
```

Reply:
agree, "for some reason" has a faint (though, I'm sure, unintended) air of eye-rolling 🙄

Reply:
Indeed. Done.